ui/helpers/utils/multichain/blockExplorer.ts

```diff
@@ -0,0 +1,39 @@
+import { getAccountLink } from '@metamask/etherscan-link';
+import { KnownCaipNamespace, parseCaipChainId } from '@metamask/utils';
+import { MultichainNetwork } from '../../../selectors/multichain';
+import { MULTICHAIN_NETWORK_TO_EXPLORER_URL } from '../../../../shared/constants/multichain/networks';
+import { normalizeSafeAddress } from '../../../../app/scripts/lib/multichain/address';
+
+export const getMultichainBlockexplorerUrl = (
+  network: MultichainNetwork,
+): string => {
+  const { namespace } = parseCaipChainId(network.chainId);
+  if (namespace === KnownCaipNamespace.Eip155) {
+    return network.network?.rpcPrefs?.blockExplorerUrl ?? '';
+  }
+
+  const multichainExplorerUrl =
+    MULTICHAIN_NETWORK_TO_EXPLORER_URL[
+      network.chainId as keyof typeof MULTICHAIN_NETWORK_TO_EXPLORER_URL
+    ] ?? '';
+
+  return multichainExplorerUrl;
+};
+
+export const getMultichainAccountLink = (
```

Comment:
This should probably be named `getMultichainAccountUrl` to match with `getMultichainBlockExplorerUrl`, WDYT?